src/App.js

```diff
@@ -354,11 +363,20 @@ export default class App extends React.Component<{}, AppState> {
                                     <PlayButton
                                         interpreterIsRunning={this.state.interpreterIsRunning}
                                         disabled={
-                                                this.state.interpreterIsRunning ||
-                                                programIsEmpty(this.state.program)}
+                                            this.state.interpreterIsRunning ||
+                                            programIsEmpty(this.state.program)}
                                         onClick={this.handleClickPlay}
                                     />
                                 </div>
+                                <div className='App__refreshButton-container'>
+                                    <RefreshButton
+                                        interpreterIsRunning={this.state.interpreterIsRunning}
+                                        disabled={
+                                            this.state.interpreterIsRunning
```

Comment:
Please place the disabled expression all on one line: `disabled={this.state.interpreterIsRunning}`